src/Controllers/PushController.js

```diff
@@ -55,6 +55,24 @@ export class PushController {
       onPushStatusSaved(pushStatus.objectId);
       return badgeUpdate();
     }).then(() => {
+      // Update audience lastUsed and timesUsed
+      if (body.audience_id) {
+        const audienceId = body.audience_id;
+
+        var updateAudience = {
+          lastUsed: { __type: "Date", iso: new Date().toISOString() },
+          timesUsed: { __op: "Increment", "amount": 1 }
+        };
+        const audienceRestQuery = new RestQuery(config, master(config), '_Audience', {objectId: audienceId});
+        audienceRestQuery.buildRestWhere().then(() => {
+          const write = new RestWrite(config, master(config), '_Audience', audienceRestQuery.restWhere, updateAudience);
```

Comment:
I believe we can use the basic RestWrite without building a restWhere query now as we only need objectId for it no?